src/cci/cas_cci.c

```diff
@@ -2811,7 +2799,7 @@ cci_get_db_version (int mapped_conn_id, char *out_buf, int buf_size)
   reset_error_buffer (&(con_handle->err_buf));
 
   API_SLOG (con_handle);
-  SET_START_TIME_FOR_QUERY (con_handle, NULL);
+  SET_START_TIME_FOR_QUERY_ONLY_CON (con_handle);
```

Comment:
I think SET_START_TIME_FOR_QUERY has no problem, but **cppcheck** seems to see this as an error. Then I think its better to rewrite it as a function 
```
void
set_start_time_for_query ()
{}
```

Reply:
I think the same. But I would like to know why this function make to macro function, 
because only macro functions have an advantage.